h/static/scripts/reducers/annotations.js

```diff
@@ -258,6 +258,13 @@ function savedAnnotations(state) {
   });
 }
 
+/** Return true if the annotation with a given ID is currently loaded. */
+function annotationExists(state, id) {
+  return state.annotations.some(function (annot) {
+    return annot.id === id;
```

Comment:
Agreed. I'd been thinking the same thing. I'd like to get https://github.com/hypothesis/client/pull/116 merged first so we can clean up the existing weirdness with the naming and non-enumerability of the tag field first.